GlobalFishingWatch/anchorages_pipeline
# Pipe tools 3 compat

* Make compatible with pipe-tools-3 and Python 3
    - test with anchorage naming
         - [x] check that runs successfully on Python 3
         - [x] check that results match Python 2
    - test with port events
         - [x] check that runs successfully on Python 3
         - [x] check that results match Python 2
    - test with port visits
         - [x] check that runs successfully on Python 3
         - [x] check that results match Python 2
* Update anchorage tables
    - update anchorage overrides
    - add Peru list
    - don't filter out inshore anchorages (per @natemiller's request).
    -  don't replace Chinese anchorage names with S2ID when in the overrides list.


**Do Not Merge Till Fully Tested**
